.github/workflows/release_process.yml

```diff
@@ -136,38 +136,6 @@ jobs:
       upload_url: ${{ needs.create_release.outputs.upload_url }}
       ver_num: ${{ needs.create_tag.outputs.new_ver}}
 
-  #
-  # WAMR_SDK
-  release_wamr_sdk_on_ubuntu_2004:
-    needs: [create_tag, create_release]
-    uses: ./.github/workflows/build_wamr_sdk.yml
-    with:
-      config_file: wamr_config_ubuntu_release.cmake
-      runner: ubuntu-20.04
-      upload_url: ${{ needs.create_release.outputs.upload_url }}
-      ver_num: ${{ needs.create_tag.outputs.new_ver}}
-      wasi_sdk_url: https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-19/wasi-sdk-19.0-linux.tar.gz
-
-  release_wamr_sdk_on_ubuntu_2204:
-    needs: [create_tag, create_release]
-    uses: ./.github/workflows/build_wamr_sdk.yml
-    with:
-      config_file: wamr_config_ubuntu_release.cmake
-      runner: ubuntu-22.04
-      upload_url: ${{ needs.create_release.outputs.upload_url }}
-      ver_num: ${{ needs.create_tag.outputs.new_ver}}
-      wasi_sdk_url: https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-19/wasi-sdk-19.0-linux.tar.gz
-
-  release_wamr_sdk_on_macos:
-    needs: [create_tag, create_release]
-    uses: ./.github/workflows/build_wamr_sdk.yml
-    with:
-      config_file: wamr_config_macos_release.cmake
-      runner: macos-latest
-      upload_url: ${{ needs.create_release.outputs.upload_url }}
-      ver_num: ${{ needs.create_tag.outputs.new_ver}}
-      wasi_sdk_url: https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-19/wasi-sdk-19.0-macos.tar.gz
```

Comment:
Same as above, maybe we need to update the script later to release the wamr-sdk.

Reply:
Ok, We update it back later.